pkg/operator/deploy/deploy.go

```diff
@@ -172,6 +180,30 @@ func (o *operator) resources() ([]runtime.Object, error) {
 						monitoringEndpoint,
 					},
 				},
+				VNetID: vnetID,
+			},
+		},
+		&cloudcredentialsv1.CredentialsRequest{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      pkgoperator.Namespace,
+				Namespace: pkgoperator.CloudCredentialsNamespace,
+			},
+			Spec: cloudcredentialsv1.CredentialsRequestSpec{
+				SecretRef: corev1.ObjectReference{
+					Namespace: pkgoperator.Namespace,
+					Name:      pkgoperator.CloudCredentialsSecret,
+				},
+				ProviderSpec: &runtime.RawExtension{
+					Raw: []byte(fmt.Sprint(`{
+	"apiVersion": "cloudcredential.openshift.io/v1",
+	"kind": "AzureProviderSpec",
+	"roleBindings": [
+		{
+		"role": "Contributor"
+		}
+	]
+}`)),
+				},
```

Comment:
I think secrets should be existing already there. Original one used by Cred Minter operator, we should consume it.